admin/js/editor.js

```diff
@@ -1,19 +1,28 @@
 /* global wp */
+const { registerBlockStyle, unregisterBlockStyle } = wp.blocks;
+const { __ } = wp.i18n;
 
 wp.domReady(() => {
-  wp.blocks.registerBlockStyle( 'core/button', {
+  registerBlockStyle('core/button', {
     name: 'secondary',
     label: 'Secondary',
     isDefault: true
   });
-  wp.blocks.registerBlockStyle( 'core/button', {
+  registerBlockStyle('core/button', {
     name: 'cta',
     label: 'CTA'
   });
-  wp.blocks.registerBlockStyle( 'core/button', {
+  registerBlockStyle('core/button', {
     name: 'donate',
     label: 'Donate'
   });
-  wp.blocks.unregisterBlockStyle('core/button', 'outline');
-  wp.blocks.unregisterBlockStyle('core/button', 'fill');
+  unregisterBlockStyle('core/button', 'outline');
+  unregisterBlockStyle('core/button', 'fill');
+
+  // Remove Take Action and Campaign covers styles for Covers block in campaigns
+  const postType = document.querySelector('form.metabox-base-form input#post_type').value;
```

Comment:
I think you can also do `wp.data.select('core').getPostType()`

Reply:
That returns `undefined`

Reply:
Right, it's `wp.data.select('core/editor').getCurrentPostType()` instead

Reply:
Ah nice, that works! Updated 🙂